ui/raidboss/data/05-shb/eureka/delubrum_reginae_savage.js

```diff
@@ -596,6 +596,364 @@ export default {
+            2: bladeValues[southCombatantBlade],
+            3: bladeValues[northCombatantBlade],
+          };
+        } else if ((southCombatantFacing === dirNum.south && bladeSides[southCombatantBlade]) ||
+          (southCombatantFacing === dirNum.north && !bladeSides[southCombatantBlade])) {
+          // South clone cleaving inside west
+          safeZone = output.northeast();
+          adjacentZones = {
+            0: bladeValues[northCombatantBlade],
+            1: bladeValues[eastCombatantBlade],
+            2: bladeValues[westCombatantBlade],
+            3: bladeValues[southCombatantBlade],
+          };
+        } else if ((southCombatantFacing === dirNum.north && bladeSides[southCombatantBlade]) ||
+          (southCombatantFacing === dirNum.south && !bladeSides[southCombatantBlade])) {
+          // South clone cleaving inside east
+          safeZone = output.northwest();
+          adjacentZones = {
+            0: bladeValues[northCombatantBlade],
+            1: bladeValues[southCombatantBlade],
+            2: bladeValues[eastCombatantBlade],
+            3: bladeValues[westCombatantBlade],
+          };
+        } else {
+          // facing did not evaluate properly
+          safeZone = output.unknown();
+          adjacentZones = null;
+        }
+
+        let currentBrand = 0;
+        let currentTemperature = 0;
+
+        if (data.currentBrand)
+          currentBrand = data.currentBrand;
+        if (data.currentTemperature)
+          currentTemperature = data.currentTemperature;
+
+        const effectiveTemperature = currentTemperature + currentBrand;
+
+        // Calculate which adjacent zone to go to, if needed
+        let adjacentZone = null;
+        if (effectiveTemperature && adjacentZones) {
+          // Find the adjacent zone that gets closest to 0
+          const calculatedZones = Object.values(adjacentZones).map((i) =>
+            Math.abs(effectiveTemperature + i));
+
+          // Use zone closest to zero as output
+          const dirs = {
+            0: output.north(),
+            1: output.east(),
+            2: output.south(),
+            3: output.west(),
+          };
+          adjacentZone = dirs[Object.values(calculatedZones).indexOf(calculatedZones.sort((a, b) =>
+            b - a).pop())];
+        } else {
+          adjacentZone = null;
+        }
+
+        // Callout safe spot and get cleaved spot if both are known
+        // Callout safe spot only if no need to be cleaved
+        if (adjacentZone)
+          data.safeZone = output.getCleaved({ dir1: safeZone, dir2: adjacentZone });
+        else if (safeZone)
+          data.safeZone = output.safeSpot({ dir: safeZone });
+        else
+          data.safeZone = null;
+      },
+      infoText: (data, _, output) => {
+        return !data.safeZone ? output.unknown() : data.safeZone;
+      },
+      outputStrings: {
+        getCleaved: {
+          en: 'Safe Spot inside ${dir1}, Get cleaved 1 square ${dir2}',
```

Comment:
```suggestion
          en: '${dir1} Safe Spot => ${dir2} for cleave',
```

Reply:
The comma was there to intentionally place a pause as it sounded better pausing between the two clauses. Does => add a pause?

Reply:
`->` or `=>` inside sentence would be replaced as "then" in various languages, e.g. replaced to "puis"  for fr players.
So make it clear when display on screen as well as in TTS.